### ISSUE TITLE
Command: Subdivide

### ISSUE THREAD

**Issue by vxlcoder** [CGCookie/retopoflow#415]
Allow user to subdivide the target (either entire mesh or selected only).

Subdivided mesh should automatically snap new verts to surface.

**Comment by carter2422** [comment]
Yes! I think this is something that should just be selection only by default, making it the same as Blender and reducing learning curve and code complexity.

**Comment by jfranmatheu** [comment]
Is this one still a thing for v4 @vxlcoder ?

**Comment by vxlcoder** [comment]
oh yeah!  i forgot about this feature.

i'll let @jlampel or @carter2422 push us off the fence with v4.

**Comment by jlampel** [comment]
Subdivide is a really important topology tool! Thankfully it is already super easy and comes for free in v4 since we can just use Blender's regular subdivide. It doesn't snap to the surface automatically, but subdividing and then snapping as two separate actions is good enough for now.
